test/e2e/handler/utils.go

```diff
@@ -521,6 +521,11 @@ func vlan(node string, iface string) string {
 	return gjson.ParseBytes(currentStateJSON(node)).Get(vlanFilter).String()
 }
 
+func kubectl(command ...string) (string, error) {
+	out, err := cmd.Kubectl(command...)
+	return out, err
+}
+
```

Comment:
Why do we need this if we have cmd.Kubectl ?

Reply:
No reason, removed this, thanks!